frontend/src/framework/internal/components/Settings/private-components/modulesList.tsx

```diff
@@ -114,15 +114,15 @@ const ModulesListItem: React.FC<ModulesListItemProps> = (props) => {
             }
         };
 
-        if (ref.current) {
```

Comment:
Line 65 needs adjustment when using `mainRef` instead of `ref`. I suggest removing `ref` and renaming `mainRef` to `ref`.